### ISSUE TITLE
[ Lien de connexion BE/CT ] A minima rendre plus visible l'info sur l'email de connexion

### ISSUE THREAD

**Issue by Octaviedebs** [MTES-MCT/Docurba#263]
Quand un BE/CT se connecte pour la 2è fois et +, le message l'invitant à aller cliquer sur le lien de connexion dans sa boite email est tout en bas et peu visible - si on reste pour l'instant sur ce format de connexion via un lien, il faudrait a minima que le message soit tout en haut + plus visible ! 

Grazie

<img width="1188" alt="Capture d’écran 2023-12-20 à 13 51 48" src="https://github.com/MTES-MCT/Docurba/assets/144041896/5e989688-522e-4844-8d8e-d94aac25b4f3">

**Comment by lerayj** [comment]
@julienzdocurba

**Comment by julienzdocurba** [comment]
Je pense qu'on peut l'alerte du design système:
![image](https://github.com/MTES-MCT/Docurba/assets/147177148/19bbbe1b-0ee4-401d-b787-8b961e424e6b)

J'ai aussi un peu changé le contenu.

**Comment by Octaviedebs** [comment]
Yes ! Si c'est rapide à faire ça peut peut être aider pas mal je pense ! 
J'ai créé la carte dans le notion backlog au fait
